lib/Sema/CSDiagnostics.h

```diff
@@ -804,6 +804,22 @@ class ContextualFailure : public FailureDiagnostic {
   }
 };
 
+class NonClassTypeToAnyObjectConversionFailure final
+    : public ContextualFailure {
+  Type RawFromType, RawToType;
+
+public:
+  NonClassTypeToAnyObjectConversionFailure(
+      const Solution &solution, Type lhs, Type rhs, ConstraintLocator *locator,
+      FixBehavior fixBehavior = FixBehavior::Error)
+      : ContextualFailure(solution, lhs, rhs, locator, fixBehavior),
+        RawFromType(lhs), RawToType(rhs){};
```

Comment:
There is no need for `;` here too. Could you please run `git-clang-format` on your changes (here is how to do it - https://github.com/apple/swift/blob/main/docs/HowToGuides/FAQ.md#how-do-i-format-my-changes).

Reply:
Oh well, at least we caught it.